scripts/airflow-entrypoint.sh

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
-airflow upgradedb
-airflow webserver
+airflow db upgrade
+airflow users create -r Admin -u admin -e admin@example.com -f Admin -l User -p admin
```

Comment:
Perhaps use eg. AIRFLOW_SECURITY_ADMIN_USER / AIRFLOW_SECURITY_ADMIN_PASSWORD or something from the .env file?